CareKitStore/CareKitStore/StoreCoordinator/OCKPersistentStoreCoordinator+Outcomes.swift

```diff
@@ -44,14 +44,13 @@ extension OCKStoreCoordinator {
             $0.anyOutcomes(matching: query)
         }
 
-        let sortDescriptor = NSSortDescriptor(
-            keyPath: \OCKCDOutcome.id,
```

Comment:
I'm assuming this is leftover from when the OCKCDOutcome.id had the the task occurrence appended to it.